lib/question.js

```diff
@@ -510,16 +510,6 @@ module.exports = {
     let questionModule, courseIssues, data, submission, grading_job;
     async.series(
       [
-        (callback) => {
-          if (question.grading_method === 'Manual') {
-            // The button to make a submission won't be available to students
-            // on manualy-graded questions; this is just here to handle the
-            // case where someone tries to be clever and manually post a response
-            // with a `grade` action.
```

Comment:
Yes, that is correct, and also because a question with other grading methods may be configured with manualPoints only. This logic has now been moved to `variants_select_submission_for_grading` instead.

https://github.com/PrairieLearn/PrairieLearn/blob/74db1d9681db56bb83d48ce888c375784251abe3/sprocs/variants_select_submission_for_grading.sql#L26-L35